server/routerlicious/packages/services/src/tenant.ts

```diff
@@ -68,7 +68,11 @@ export class TenantManager implements core.ITenantManager {
             });
         };
         const defaultHeaders = getDefaultHeaders();
-        const baseUrl = `${details.data.storage.internalHistorianUrl}/repos/${encodeURIComponent(tenantId)}`;
+        let internalHistorianUrl = this.defaultInternalHistorianUrl;
```

Comment:
```suggestion
        const internalHistorianUrl = this.defaultInternalHistorianUrl ?? details.data.storage.internalHistorianUrl;
```

Reply:
Thanks for the suggestion. I might also check the empty string for the defaultInternalHistorianUrl.

Reply:
Initially, the code was assigning `this.defaultInternalHistorianUrl` to `internalHistorianUrl `. Then you would check `internalHistorianUrl` - which was the same as checking `this.defaultInternalHistorianUrl` - to see if it was falsy, and if yes, you would overwrite it with `details.data.storage.internalHistorianUrl`.

Basically, that means: is `this.defaultInternalHistorianUrl` valid? If yes, use it. if not, use `etails.data.storage.internalHistorianUrl`. And that's what the suggestion above does

Reply:
I think you are right, `??` just checks for undefined and not generally for falsy values. Thanks for catching that. I think in that case, to check both for undefined and empty string, we would likely have to use `||` instead, as Zach mentioned in a different comment. I tested it here: [link](https://www.typescriptlang.org/play?#code/GYVwdgxgLglg9mABBAFgUwgawDJwOYwQAUcATjAWAIYA2AXIgM5Tlh4A0icAbmqeQBM0AeXKVaAfgbNWeAJSIA3gChEaxKTRQQpJDz6CRYmNRqIAPua7HTAbmUBfZcogJGcGmgB0NfEVQYOPiERABEZBQmtKGc4bz8MEKikaahcnL2Lm4e3r54-uhYuATE4TbRsWkZzq5g7p4+fgFFwaUR4jRVtkA)

Reply:
Got it. Thanks for check, change it to `||`